tests/jvm/src/test/scala/io/iteratee/files/TryTests.scala

```diff
@@ -0,0 +1,26 @@
+package io.iteratee.files
+
+import io.iteratee.{ Iteratee, Enumerator }
+import io.iteratee.tests.BaseSuite
+import scala.util.{ Success, Try }
+import java.io.File
+
+class TryTests extends BaseSuite {
```

Comment:
`it` is the conventional directory for integration tests—it's treated specially by SBT, etc. It probably doesn't matter much in this case, but in general I'd put tests there if they do any I/O.